### PR TITLE
RHIDP-3022: Document how to track changes to catalog database

### PR DESCRIPTION
**Pull request by linfraze**
**Version(s):**
1.2.2

**Issue:**
https://issues.redhat.com/browse/RHIDP-3022

**Link to docs preview:**
https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-371/getting-started-rhdh/#con-audit-log-track-changes-catalog_assembly-audit-log

**Reviews:**
- [x] SME: @kim-tsao @schultzp2020 
- [ ] QE: N/A - conceptual info only
- [x] Docs review: @hmanwani-rh 

**Additional information:**
Blurb about tracking catalog database changes has also been added to the [RNs draft doc](https://docs.google.com/document/d/1XMhZUKQ0SOMr86mdLiyVTSU0kg4RqtJb8lfoFZ9QRFQ/edit?usp=sharing)

### CODE REVIEW

**Comment by schultzp2020** [comment]
lgtm

File: modules/getting-started/con-audit-log-track-changes-catalog.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:_mod-docs-content-type: CONCEPT
+[id="con-audit-log-track-changes-catalog_{context}"]
+= Tracking changes in the {product-short} catalog database
```
Comment (hmanwani-rh) [inline modules/getting-started/con-audit-log-track-changes-catalog.adoc:3]: Headings that start with gerunds indicate procedural modules. I recommend changing the heading to:
```suggestion
= Changes in the {product-short} catalog database
```

File: assemblies/assembly-audit-log.adoc [hunk 2]
```diff
@@ -35,3 +35,5 @@ Audit logs are not forwarded to the internal log store by default because this d
 * For more information, see link:https://docs.openshift.com/container-platform/latest/observability/logging/cluster-logging.html[{ocp-brand-name}  - About Logging]
 
 include::modules/getting-started/con-audit-log-config.adoc[leveloffset=+1]
```
Comment (hmanwani-rh) [inline assemblies/assembly-audit-log.adoc:37]: Similarly, 'con-audit-log-config.adoc' is a conceptual module, but its heading starts with a gerund, which typically indicates a procedural module. Should we change that as well, or is a separate Jira needed?

File: modules/getting-started/con-audit-log-track-changes-catalog.adoc [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+In {product} {ocp-version-min} and later, changes to the catalog database are forwarded to a central log management system, such as ElasticSearch or Splunk by default. Tracking changes that add, remove, or update data in the catalog database informs administrators of user actions to help ensure accountability and transparency.
```
Comment (hmanwani-rh) [inline modules/getting-started/con-audit-log-track-changes-catalog.adoc:5]:
```suggestion
In {product} {ocp-version-min} and later, changes to the catalog database are forwarded to a central log management system, such as ElasticSearch or Splunk by default. Changes that add, remove, or update data in the catalog database inform administrators of user actions, ensuring accountability and transparency.
```

**Review by hmanwani-rh — COMMENTED** [review]
@linfraze Added a few comments below. Please take a look and let me know if you've any question. 

Quick question: If the tracking changes section lacks detailed information, do you think we should include this information in the introduction paragraph instead? I’m not sure if having a separate section for such minimal content is necessary. What do you think?

File: assemblies/assembly-audit-log.adoc (hunk 2 above)
Comment (linfraze) [reply assemblies/assembly-audit-log.adoc:37]: You are right about procs generally beginning with gerunds and cons generally beginning with nouns, but I don't think this is a necessary change. It is also highly likely that this assembly will provide some procedural steps in the future, we just aren't there yet.

**Comment by linfraze** [comment]
> @linfraze Added a few comments below. Please take a look and let me know if you've any question.
> 
> Quick question: If the tracking changes section lacks detailed information, do you think we should include this information in the introduction paragraph instead? I’m not sure if having a separate section for such minimal content is necessary. What do you think?

Sure, I will go ahead and incorporate this content into assembly, but leave the con module in the repo in case it is needed in the future. This will also resolve the gerund heading for now.

**Pull request merged**